app/assets/javascripts/iframe-saver.coffee

```diff
@@ -9,11 +17,13 @@ class IFrameSaver
   constructor: ($iframe, $data_div, $delete_button) ->
     @$iframe = $iframe
     @$delete_button = $delete_button
+    @save_state = $data_div.data('save-state');
     @interactive_run_state_url = $data_div.data('interactive-run-state-url') # get and put our data here.
     @collaborator_urls = $data_div.data('collaborator-urls')
     @auth_provider = $data_div.data('authprovider') # through which provider did the current user log in
     @user_email = $data_div.data('user-email')
     @logged_in = $data_div.data('loggedin') # true/false - is the current session associated with a user
+    @authoredState = getAuthoredState($data_div) # state / configuration provided during authoring
```

Comment:
OK I finally figured this out. :) `save_state` is a boolean from the interactive, it is not the state from the interactive_run_state. I was reading it as "save*d* state".
 
It would be good to clear this up. However changing the name of `save_state` would require changing 27 references to save_state and dealing with the export and import code, so old activity exports would still import. 

I'd suggest just changing it in javascript and the dom so it is more clear, and we don't have to change the export and import code. Perhaps `enable_learner_state`.